src/components/MemberComponent.js

```diff
@@ -114,10 +116,17 @@ const MemberComponent = ({ displayPopup }) => {
     setState({ memberList: [...state.memberList, member] });
   };
 
-  const onDeleteItem = (item) => {
+  const onDeleteItem = async (item) => {
     setState({
       memberList: state.memberList.filter((member) => member !== item),
     });
+    if(sessionStorage.getItem('user') && item.email===(JSON.parse(sessionStorage.getItem('user'))).email){
+      await updateProject(state.memberList.filter((member) => member !== item)) //Force the update as setState wont act immediately
+
+      sessionStorage.removeItem('project');
+      sessionStorage.removeItem('sprint');
+      history.push('/home'); //Bit janky with projects still being selected in the top bar
+    }
```

Comment:
you can use `history.go(0)` to reload the page, we shouldn't have to redirect on deletion